cumulus/polkadot-omni-node/README.md

```diff
@@ -39,12 +39,14 @@ cargo install staging-chain-spec-builder --locked
 
 ### 2. Generate a chain spec
 
-Omni Node expects for the chain spec to contain parachains related fields like `relay_chain` and `para_id`.
-These fields can be introduced by running [`staging-chain-spec-builder`](https://crates.io/crates/staging-chain-spec-builder)
-with additional flags:
+Omni Node expects for the chain spec to contain a JSON key called `relay_chain`. This is usually set
+to a chain id, mapped out by the [load_spec](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/cli/src/command.rs#L77)
+to a chain spec object corresponding to the picked relaychain.
```

Comment:
Would you please rephrase this sentence, not sure if I understand:
"This is usually set
to a chain id, mapped out by the [load_spec](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/cli/src/command.rs#L77)
to a chain spec object corresponding to the picked relaychain."

I also think we should not give references to the code (also in master the code can be moved). Maybe better name the CLI option?

Reply:
`load_spec` function is part of some CLI commands, e.g. `Subcommand::ExportChainSpec`. Referencing their names  (as user see them on the CLI) would be more readable then code maybe.

Reply:
> Referencing their names (as user see them on the CLI) would be more readable then code maybe.

As a matter of fact, we might be able to export some of that code into a "doc" oriented section, with docify. I'll investigate. Maybe this is what you had in mind?

Reply:
What I had in mind is not referencing the code in the readme of the tool ;). How this hepls to onboard new users? We should simplify, and lower the entry barrier. Up to you.

Reply:
> What I had in mind is not referencing the code in the readme of the tool ;). How this hepls to onboard new users? 

We're on the same page here. I think we can definitely do better. Will ping when I have something to show

Reply:
Added the actual chain ids literally in the docs, except `versi`, which I haven't seen it used that much in the context of testing of the parachains: [81c2a1b](https://github.com/paritytech/polkadot-sdk/pull/8651/commits/81c2a1bfc19dabf57ec9cccdd01e600902caa81f). And totally unrelated, I also thought there is a gap when considering local testing, where `westend-local/rococo-local` are missing from the default `polkadot-omni-node` build, while they are enabled for `polkadot-parachain`, so decided to mention and enable them. I don't plan to do more docs for chain ids for now.

Resolving this but please lmk if you have feedback. Would be happy to reopen, discuss and incorporate.